pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+<h4> Issue(Describe in 2-3 lines what the current issue or background is) : </h4>
+<br>
+<h4> Fix or Changes done(Describe in 2-3 lines changes or fix made) : </h4>
+<br>
+<h4> Bug ID(if any) : </h4>
+<br>
+<h4> Public Documentation(if any) : </h4>
+<br>
+<h4> Testing Conducted(Test Cases with scenario and description - must have 1 positive and 1 negative scenario) : </h4>
```

Comment:
Can we include a checkbox to ensure all unit test cases are passing? Taking reference from 'java-code-samples' repository.